core/src/test/scala/org/apache/spark/scheduler/OutputCommitCoordinatorIntegrationSuite.scala

```diff
@@ -47,11 +46,12 @@ class OutputCommitCoordinatorIntegrationSuite
 
   test("exception thrown in OutputCommitter.commitTask()") {
```

Comment:
since we changed this test, let's put the JIRA ID in the test name

Reply:
> since we changed this test, let's put the JIRA ID in the test name

Done